.gitignore

```diff
@@ -27,3 +27,4 @@ chrome-user-data
 .idea
 *.iml
 .vscode
+yarn.lock
```

Comment:
Yes, please remove this, we explicitly made sure to check this in https://github.com/facebook/react/pull/7957